articles/types/jsdoc.md

```diff
@@ -1,35 +1,37 @@
 # JsDoc support
 
 Programs written in JavaScript and TypeScript can be annotated with (JsDoc comments)[http://usejsdoc.org/index.html]
-to give type information and quick-info descriptions of identifiers.
+to give type information and quickinfo descriptions of identifiers.
 
-In the following example, the text between the opening `/**` and closing `/*` are considered JsDoc comments.
+In the following example, the text between the opening `/**` and closing `*/` are considered JsDoc comments.
 
 ```js
 /**
- * @param {"cat" | "dog"} kind  The type of animal to create.
- * @param name {string}
- * @param {number} weight The weight, in pounds, of animal.
- * @param {...string} extraProps Any extra properties to tack on the animal.
- * @return {{[index: string]: string | number}} Any extra properties to tack on the animal.
+ * Creates a new animal object.
+ * @param {string} name The name of the new animal.
+ * @param {number} weight
+ * @param {"cat" | "dog"} kind
+ * @return {{name: string, weight: number, kind: "cat" | "dog"}} A brand new animal!
  */
-function createAnimal(kind, name, weight, ...extraProps)
+function createAnimal(name, weight, kind)
 {
     // ...
 }
 ```
 
 As you can see, we can prefix sections in JsDoc with terms such as `@param` and `@return` to describe the behavior of the `createAnimal` function.
 In JsDoc, terms beginning with `@` have special meanings, and are called *tags*. The text between curly braces are descriptions of *types*,
-which can be used by the developer to check that they are using a variable correctly -- they can go either before or after the construct's *name*.
+which can be used by the developer to check that they are using a variable correctly &mdash;  they should precede the *name* of the construct being described, if applicable.
```

Comment:
double-space is wrong here, although I don't know if it will show up in the HTML output.